decidim/decidim
# Upgrade D3 to 7.8.4

#### :tophat: What? Why?
This PR updates D3 from `5.4.0` to `7.8.4`.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #8586

#### Testing
1. Make sure the metrics displayed in admin are ok 
2. Make sure the assemblies organizational chart is ok.

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
